docs/rules/style/messy-rule.md

```diff
@@ -0,0 +1,57 @@
+# messy-rule
```

Comment:
I wonder if 'disjointed' is would be more specific here? Fragmented might be another option.

Reply:
Either way you'd need to read the rule docs to learn what it is about, so I think "messy" works well to catch the user's attention :)